tsl/test/expected/compression.out

```diff
@@ -1274,7 +1274,7 @@ SELECT compress_chunk(c) FROM show_chunks('stattest') c;
 SELECT approximate_row_count('stattest');
  approximate_row_count 
 -----------------------
-                     0
+                    26
```

Comment:
You might want to add a test that compares this with the actual number of rows counted explicitly. For cases where you have serial execution, you should get the same value.

The test also seems to be missing some cases where you have a mix of uncompressed and compressed rows in a chunk, so might be good to verify that.